src/App.tsx

```diff
@@ -1,61 +1,115 @@
+import React, { useState } from 'react';
 import './App.scss';
+import { TodoList } from './components/TodoList';
+import { Todo } from './Types/Todo';
+import { User } from './Types/User';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+const findUserById = (userId: number) => (
+  usersFromServer.find(user => user.id === userId) as User
+);
+
+const allTodos: Todo[] = todosFromServer.map(todo => ({
+  ...todo,
+  user: findUserById(todo.userId),
+}));
```

Comment:
you could specify more with its naming what it does. for one who doesnt know the task it might not be clear.

e.g.: const todosWithUsers ...